### ISSUE TITLE
React and ant-design stack

### ISSUE THREAD

**Issue by djkloop** [withastro/astro#6806]
### What version of `astro` are you using?

2.2.1

### Are you using an SSR adapter? If so, which one?

None

### What package manager are you using?

npm

### What operating system are you using?

windows

### What browser are you using?

Edge

### Describe the Bug

I need to use ant-design in the page, and also need to configure the theme style, but when I was in the local, he was always stuck, and then I tried to reproduce this issue on stackblitz, but the page crashed directly.

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/github-bjlru4?file=src/pages/index.astro

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by djkloop** [comment]
git demo 

https://github.com/djkloop/astro-antd-issue

**Comment by bluwy** [comment]
It's showing fine for me, can you try with this config in `package.json`?

```json
"pnpm": {
  "overrides": {
    "vite": "^4.3.0-beta.3"
  }
}
```

Make sure to `pnpm i` again after it's set. I think you're seeing performance issue in Vite and this has been fixed in the Vite 4.3 beta.

**Comment by djkloop** [comment]
Thank you, after this configuration is changed, it can indeed run. @bluwy
